packages/jsoo-react/jsoo-react.0.1/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings to ReactJS for js_of_ocaml, including JSX ppx"
+maintainer: ["Javier Chávarri <javier.chavarri@gmail.com>"]
+authors: ["Javier Chávarri <javier.chavarri@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/ml-in-barcelona/jsoo-react"
+bug-reports: "https://github.com/ml-in-barcelona/jsoo-react/issues"
+depends: [
+  "dune" {>= "2.7" & >= "2" & < "4"}
```

Comment:
```suggestion
  "dune" {>= "2.7"}
```
This can also be fixed by removing dune as a dependency in the dune-project file (it is added automatically). I would not add an upper bound here, dune tends to be backward compatible and the CI thoroughly checks this upon release